front/src/components/boxs/weather/WeatherBox.jsx

```diff
@@ -176,7 +176,7 @@ const WeatherBox = ({ children, ...props }) => (
                   color: 'grey'
                 }}
               >
-                {props.units === 'si' ? 'km/h' : 'm/h'}
+                {props.units === 'metric' ? 'm/s' : 'm/h'}
```

Comment:
It would be better to use constants and i18n instead of hard coded translations here.

Reply:
Fixed :)